packages/core/src/colors/palette.ts

```diff
@@ -25,4 +26,7 @@ export interface Palette {
 
   disabledBackground: string;
   disabledForeground: string;
+
+  error: string;
+  errorContrast: string;
```

Comment:
@serifine what is your opinion about these variables and their names ?

Following your advice for `controlLabelColor`, should we go for something easier to catch such as `errorColor` and `errorContrastColor`?